sycl/test/reduction/reduction_nd_ext_type.hpp

```diff
@@ -0,0 +1,89 @@
+// UNSUPPORTED: cuda
```

Comment:
This is a header file, so I think that all UNSUPPORTED and RUN: lines don't make sense here. Could you please remove them.

Reply:
Right. Fixed. Thank you!